content/sensu-go/5.2/_index.md

```diff
@@ -1,22 +1,21 @@
 ---
 title: "Sensu Go"
-description: "Sensu is the open source monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures."
-date: 2018-02-05T06:06:31-07:00
+description: "Sensu is the open source monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures. For enterprise-class features and support for Sensu Go, try Sensu Enterprise."
 weight: -100
 menu: "sensu-go-5.2"
 version: "5.2"
 product: "Sensu Go"
 layout: "single"
 ---
 
-<iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-go&type=star&count=true" frameborder="0" scrolling="0" width="87px" height="20px"></iframe> | [Share your feedback][2]
+<iframe src="https://ghbtns.com/github-btn.html?user=sensu&repo=sensu-go&type=star&count=true" frameborder="0" scrolling="0" width="87px" height="20px"></iframe> | [Learn about Sensu Enterprise][28]
 
 Sensu is the open source monitoring event pipeline built to reduce operator burden and make developers and business owners happy.
 Started in 2011, Sensu’s flexible approach solves the challenges of monitoring hybrid-cloud and ephemeral infrastructures with scalable, automated workflows and integrations with tools you already use.
 **Get started now and feel the #monitoringlove:** [Learn Sensu Go][1].
 
 Sensu Go is the latest version of Sensu, written in Go and designed to be more portable, easier and faster to deploy, and (even more) friendly to containerized and ephemeral environments.
-But above all, it was designed to support the features and functionality you’ve come to know and love about Sensu.
+For enterprise-class features and support for Sensu Go, try [Sensu Enterprise][28].
```

Comment:
I'm wondering if we need "for Sensu Go" in this sentence -- it kind of sounds like Sensu Go is what's being supported. With the paragraph above, I think it's clear we're talking about Sensu Go. Thoughts?

Reply:
> "it kind of sounds like Sensu Go is what's being supported."

This was my intention, but it seems like it's creating some confusion. Let's chat about this synchronously if you have time this week.

Reply:
Options:
1. For enterprise-class support, LDAP authentication, and support for industry-standard tools, check out Sensu Enterprise.
2. For enterprise-class features and support for your Sensu Go installation, try Sensu Enterprise.